src/VisualStudio/CSharp/Test/EventHookup/EventHookupTestState.cs

```diff
@@ -12,19 +12,23 @@
 using Microsoft.CodeAnalysis.Editor.UnitTests.Extensions;
 using Microsoft.CodeAnalysis.Shared.TestHooks;
 using Microsoft.CodeAnalysis.Test.Utilities;
-using Microsoft.VisualStudio.Composition;
 using Roslyn.Utilities;
 using Xunit;
 
 namespace Microsoft.CodeAnalysis.Editor.CSharp.UnitTests.EventHookup
 {
     internal sealed class EventHookupTestState : AbstractCommandHandlerTestState
     {
+        // TODO: It seems that we can move EventHookupSessionManager to EditorFeatures.
+        private static readonly TestComposition s_composition = EditorTestCompositions.EditorFeaturesWpf.AddParts(
+            typeof(EventHookupCommandHandler),
+            typeof(EventHookupSessionManager));
```

Comment:
❔ If you're testing features from Microsoft.VisualStudio.LanguageServices.CSharp, why not just use the composition that includes it?

Reply:
We could use the VS composition. As the TODO says, the implementation doesn't seem have VS dependencies and could be potentially moved to EditorFeatures layer.

Reply:
Filed https://github.com/dotnet/roslyn/issues/46280 to follow up.